doc/docker/jenkinsnode/Dockerfile

```diff
@@ -0,0 +1,95 @@
+FROM debian:stretch
+
+ARG SWIGVERSION="3.0.11"
+
+# install dependencies
+RUN apt-get update && apt-get -y upgrade
+RUN apt-get -y install \
+  apt-utils \
```

Comment:
Is this a dep?

Reply:
apt-utils? I saw some output while the container was built where it complained that this package is missing, but i think it works anyway, so i'll remove it if not a must

debconf: delaying package configuration, since apt-utils is not installed

Reply:
> package is missing

Maybe it needs to be installed on the host?

Reply:
looks like its not needed at all